### PR TITLE
[json-rpc] fix upgrade event module name

### PR DESCRIPTION
**Pull request by xli**
## Motivation

Added integration test to make sure we convert onchain upgrade event correct.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

yes

## Test Plan

integration test

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/libra/developers.libra.org, and link to your PR here.)

### CODE REVIEW

**Comment by libra-action** [comment]
**This PR may have modified JSON-RPC server code.**

Breaking changes policy:

1. Chaning JSON-RPC method signatures or removing/modifying fields
   in the response in /V1 will not be accepted.
2. Adding fields is ok.
3. Adding a new JSON-RPC method is ok.

Please ensure you have also done the following to update the docs:

1. Update json-rpc/API-CHANGELOG.md.
2. Add/update type/method documentation under json-rpc/docs.

File: types/src/account_config/events/upgrade.rs [hunk 1]
```diff
@@ -23,6 +23,6 @@ impl UpgradeEvent {
 }
 
 impl MoveResource for UpgradeEvent {
-    const MODULE_NAME: &'static str = "LibraWriteSetManager";
```
Comment (runtian-zhou) [inline types/src/account_config/events/upgrade.rs:26]: Thanks for catching that!

File: json-rpc/tests/integration_test.rs [hunk 2]
```diff
@@ -598,6 +601,45 @@ fn create_test_cases() -> Vec<Test> {
...
+                );
+                let result = env.submit_and_wait(txn);
+                let version = result["version"].as_u64().unwrap();
+                assert_eq!(
+                    result["events"],
+                    json!([
+                        {
+                            "data":{
+                                "type": "upgrade",
+                                "write_set": "01c4c63f80c74b11263e421ebf8486a4e30901217da6c6b3e19f180104cafed00d00"
```
Comment (runtian-zhou) [inline json-rpc/tests/integration_test.rs:622]: Do we want to add an extra test to make sure the writeset here is equal to the lcs serialized common writeset?

**Review by runtian-zhou — APPROVED** [review]
Awesome!!

**Pull request closed without merging**